tests/test_c_interpolation_tables.py

```diff
@@ -885,9 +872,7 @@ def test_Nion_conditional_tables(name, R, mini, intmethod, plt):
         INHOMO_RECO=True,
         USE_TS_FLUCT=True,
     )
-    lib.Broadcast_struct_global_PS(up(), cp())
-    lib.Broadcast_struct_global_UF(up(), cp())
-    lib.Broadcast_struct_global_IT(up(), cp(), ap(), fo())
+    lib.Broadcast_struct_global_all(up(), cp(), ap(), fo())
```

Comment:
This is not your task necessarily (certainly not in this PR) but I'd love to have a nicer lower level wrapper that just exposes these functions directly in Python style, so you could call like

````python
broadcast_structs(up, cp, ap, fo)
```

Reply:
do you mean just adding something like the following to the wrapper? these functions are already exposed

```
def broadcast_inputstructs(up,cp,ap=None,fo=None):
    if not isinstance(up,UserParams) or not isinstance(cp,CosmoParams):
        raise ValueError
    if isinstance(ap,AstroParams) and isinstance(fo,FlagOptions):
        lib.Broadcast_struct_global_all(up(), cp(), ap(), fo())

    lib.Broadcast_struct_global_noastro(up(), cp())
```

Reply:
Exactly.